pkg/network/node/ovscontroller.go

```diff
@@ -333,35 +335,32 @@ func (oc *ovsController) getPodDetailsBySandboxID(sandboxID string) (int, net.IP
 	strports, err := oc.ovs.Find("interface", "ofport", "external-ids:sandbox="+sandboxID)
 	if err != nil {
 		return 0, nil, err
-	} else if len(strports) == 0 {
-		return 0, nil, fmt.Errorf("failed to find pod details from OVS flows")
-	} else if len(strports) > 1 {
-		return 0, nil, fmt.Errorf("found multiple ofports for sandbox ID %q: %#v", sandboxID, strports)
 	}
-	ofport, err := strconv.Atoi(strports[0])
+	strIDs, err := oc.ovs.Find("interface", "external-ids", "external-ids:sandbox="+sandboxID)
```

Comment:
getPodDetailsBySandboxID() is called by both UpdatePod() and TeardownPod() which is expected to be called frequently in some cases (e.g: online). If we change ovs.Find() interface to take multiple columns, then we could get strports and strIDs in one call.

Reply:
I'm not sure it's *that* frequent, relative to how expensive the operation is ("not very"). All the other callers of ovs.Find want to use a single column, so it would be a little annoying to force them to pass an array of columns names and then get back an array of arrays as the answer...

(Note that we're switching from "1 ovs-vsctl call plus 1 ovs-ofctl call" to "2 ovs-vsctl calls", so we're not making things worse, just failing to make them better.)